2023-risks/M8: Insufficient Cryptography

```diff
@@ -0,0 +1,106 @@
+---
+
+layout: col-sidebar
+title: "M8: Insufficient Cryptography"
+---
+
+# Threat Agents
+
+**Application Specific**
+
+Individuals who possess physical proximity to data that has been inadequately encrypted or mobile malware that operates on behalf of an adversary can compromise the security of the system.
```

Comment:
What would be reasons for threat agents to become active? What motivates them? I would list a few examples.

E.g., accessing valuable key material that is encrypted in a backup of the app stolen from a popular person.